docs/user/about/index.rst

```diff
@@ -39,3 +37,43 @@ This allows us to give back to the communities and projects that we support and
 
 We are proud about the way we manage our company and products,
 and are glad to have you on board with us in this :doc:`great documentation journey </story>`.
+
+A reference to our organization
+-------------------------------
+
+As a company that works with documentation,
+we enjoy using documentation tools for maintaining most of our information.
+For instance,
+we find that Git and documentation tools are extremely efficient for maintaining policies and legal documents.
+
+.. seealso::
+
+   `Our website <https://about.readthedocs.com>`__
+      You won't find everything explained here.
+      The most general-purpose information about Read the Docs is available on our main website.
+
+Here is a list of information about our organization that we currently maintain using documentation tools:
```

Comment:
This is really oddly specific? I think it's awkard to try and explain why we're using "docs tools" for this twice, instead of just referencing it?

```suggestion
```

Reply:
I agree, sorry about dumping an experiment into the PR. I was sketching how it would look if we tried to include the user into a "here is what we enjoy using documentation tools for" (such as legal documents and policies) -- but it became very meta. Could perhaps figure in an admonition box some other sunny day :)

Reply:
(it was a last minute commit&push before signing off Friday)